CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 *
 
+# 2.0.0
+
+* Add "script" and "style" to the default set of [clean content tags]
+* Make all iterator-accepting APIs use `IntoIterator` and `Borrow`, so that you can pass slices directly to them.
+
+[clean content tags]: https://docs.rs/ammonia/1.2.0/ammonia/struct.Builder.html#method.clean_content_tags
```

Comment:
You might want to link to 2.0.0 directly ?

Reply:
Yeah...